src/a11y-dialog.js

```diff
@@ -1,283 +1,237 @@
 import focusableSelectors from 'focusable-selectors'
 
-var TAB_KEY = 'Tab'
-var ESCAPE_KEY = 'Escape'
+const TAB_KEY = 'Tab'
+const ESCAPE_KEY = 'Escape'
 
 /**
- * Constructor to instantiate a dialog
- *
- * @class
- * @constructor
- * @param {Element} element
+ * @typedef { import ('./a11y-dialog').A11yDialog } A11yDialogType
+ * @typedef { import('./a11y-dialog').EventType } EventType
+ * @typedef { import('./a11y-dialog').EventHandler } EventHandler
+ * @typedef { import('./a11y-dialog').ListenersRecord } ListenersRecord
```

Comment:
So what’s any of this? 😹

Reply:
Force of habit? 🤣 This is JSDoc syntax that imports the types defined in the d.ts file, so they can be referenced with the @param keyword. It’s not necessary since we know TypeScript is coming soon, though